src/services/accessValidation.js

```diff
@@ -19,23 +19,35 @@ const logContext = {
  * @returns {Promise<any>} - returns a promise
  */
 export default function findOrCreateUser(data) {
-  return sequelize.transaction((transaction) => User.findOrCreate({
+  return sequelize.transaction((transaction) => User.findOne({
```

Comment:
This is done to check whether a user was found by `hsesUserId `or `hsesUsername `for logging purposes